src/Repository/Doctrine/AccountRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Adshares\AdsOperator\Repository\Doctrine;
+
+use Adshares\AdsOperator\Document\Account;
+use Adshares\AdsOperator\Repository\AccountRepositoryInterface;
+
+class AccountRepository extends BaseRepository implements AccountRepositoryInterface
+{
+    /**
+     * @return array
+     */
+    public function availableSortingFields(): array
+    {
+        return [
+            'id',
+            'time',
+        ];
+    }
+
+    /**
+     * @param string $accountId
+     * @return Account|null
+     * @throws \Doctrine\ODM\MongoDB\LockException
+     * @throws \Doctrine\ODM\MongoDB\Mapping\MappingException
+     */
+    public function getAccount(string $accountId):? Account
+    {
+        /** @var Account $account */
+        $account = $this->find($accountId);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$account". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1488)